### ISSUE TITLE
python webui.py启动一直卡主

### ISSUE THREAD

**Issue by leoterry-ulrica** [chatchat-space/Langchain-Chatchat#736]
**问题描述 / Problem Description**
# python webui.py
启动一直卡在这个地方：
INFO  2023-06-29 08:37:58,000-1d: Load pretrained SentenceTransformer: ./langchain-ChatGLM/text2vec-large-chinese
WARNING 2023-06-29 08:37:58,002-1d: No sentence-transformers model found with name ./langchain-ChatGLM/text2vec-large-chinese. Creating a new one with MEAN pooling.
WARNING 2023-06-29 08:38:07,536-1d: The dtype of attention mask (torch.int64) is not bool

**环境信息 / Environment Information**
- langchain-ChatGLM 版本/commit 号：langchain-ChatGLM master
- 是否使用 Docker 部署（是/否）：否
- 使用的模型（ChatGLM-6B / ClueAI/ChatYuan-large-v2 等）：ChatGLM-6B-int4
- 使用的 Embedding 模型（GanymedeNil/text2vec-large-chinese 等）：GanymedeNil/text2vec-large-chinese
- 操作系统及版本 / Operating system and version: windows10
- Python 版本 / Python version: 3.8.1
- 其他相关环境信息 / Other relevant environment information: CPU，16G内存

**Comment by GordenChoy** [comment]
我也是  模型加载成功后 就一直卡住  没动静了

**Comment by GordenChoy** [comment]
等了几分钟有反应了  提示了一个错误：ERROR 2023-06-30 17:35:06,753-1d: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
后面显示running了  端口号出来了

**Comment by zhubao315** [comment]
网络问题。如果网络通，重启几次试试。

**Comment by leoterry-ulrica** [comment]
> 网络问题。如果网络通，重启几次试试。

加载本地模型，而且网络是通的（含科学上网）

**Comment by leoterry-ulrica** [comment]
> 等了几分钟有反应了 提示了一个错误：ERROR 2023-06-30 17:35:06,753-1d: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')) 后面显示running了 端口号出来了

最终我的依然没显示running

**Comment by panglinhao-1** [comment]
俺也一样

**Comment by flowbywind** [comment]
我的是向量模型文件没改为本地路径，所以每次执行网络下载因为网络不好报错。需要修改model_config里向量那个文件embedding_model_dict里text2vec值到本地位置 "text2vec": "/opt/*****/text2vec-large-chinese", 同样LLM的文件也要改到本地路径
